### PR TITLE
English translation

### PR DESCRIPTION
**Pull request by Plenet-OTAKU**
Full translation in english

### CODE REVIEW

**Review by felix-d1strict — CHANGES_REQUESTED** [review]
You should remove the "non-informal" version in some phrases like:
`here {if LANGUAGE_USE_INFORMAL_VARIANT}you{/if}can specify a different address for the TeamSpeak server. This is displayed instead of the query address.`

"here can specify [...]" sounds a bit weird. This is just an example, there are few more phrases.


Edit: Another example `{if LANGUAGE_USE_INFORMAL_VARIANT}Choose{else}Choose you{/if} the TeamSpeak connection for the viewer.`

"Choose you the TeamSpeak connection [...]" sounds weird too.

File: language/en.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<language xmlns="https://www.woltlab.com" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://www.woltlab.com https://www.woltlab.com/XSD/vortex/language.xsd" languagecode="en">
+<!-- 
+Translation provided by Plenet_OTAKU
+-->
+	<category name="wcf.acp.option">
+		<item name="wcf.acp.option.category.teamspeak_viewer"><![CDATA[Viewer]]></item>
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_identity"><![CDATA[TeamSpeak connection]]></item>
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_identity.description"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Choose{else}Choose you{/if} the TeamSpeak connection for the viewer.]]></item>
```
Comment (Hanashi) [inline language/en.xml:9]: Please don't use informal in english language file.

File: language/en.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_cache_interval"><![CDATA[Cache interval (in seconds)]]></item>
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_cache_interval.description"><![CDATA[Every how many seconds you want the current view to be saved in the cache.]]></item>
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_show_data"><![CDATA[Show connection data]]></item>
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_show_data.description"><![CDATA[If this option is active, the data for connecting to the server is displayed in the server viewer.]]></item>
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_address"><![CDATA[Alternative address]]></item>
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_address.description"><![CDATA[here {if LANGUAGE_USE_INFORMAL_VARIANT}you{/if}can specify a different address for the TeamSpeak server. This is displayed instead of the query address.]]></item>
```
Comment (Hanashi) [inline language/en.xml:15]: Please don't use informal in english language file.

File: language/en.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_port"><![CDATA[Alternative Port]]></item>
+		<item name="wcf.acp.option.hanashi_teamspeak_viewer_port.description"><![CDATA[here {if LANGUAGE_USE_INFORMAL_VARIANT}you{/if}can specify a different address for the TeamSpeak server. This is displayed instead of the query address.]]></item>
```
Comment (Hanashi) [inline language/en.xml:17]: Please don't use informal in english language file.

File: language/en.xml (hunk 1 above)
Comment (Hanashi) [inline language/en.xml:9]: Please don't use informal in english language file.

File: language/en.xml (hunk 2 above)
Comment (Hanashi) [inline language/en.xml:15]: Please don't use informal in english language file.

File: language/en.xml (hunk 3 above)
Comment (Hanashi) [inline language/en.xml:17]: Please don't use informal in english language file.

**Review by Hanashi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**